src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2427,6 +2427,21 @@ limitations under the License.
             <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"/>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-thread-border-router</name>
+        <domain>HRAP</domain>
+        <typeName>Matter Thread Border Router</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0xFFF10011</deviceId><!-- TODO: ID-TBD -->
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters lockOthers="true">
```

Comment:
The spec I am looking at has "Network Commissioning" as a required cluster for this device type, right?

Reply:
The required clusters have been changed after the SPEC PR merged.